src/main/kotlin/de/bund/digitalservice/useid/identification/IdentificationSessionController.kt

```diff
@@ -23,56 +23,67 @@ import org.springframework.http.HttpStatus
     private val eidServiceConfig: EidServiceConfiguration
 ) {
     private val log = KotlinLogging.logger {}
-    private val tcTokenCallsSuccessfulCounter: Counter = Metrics.counter(METRIC_NAME_EID_SERVICE_REQUESTS, "method", "get_tc_token", "status", "200")
-    private val tcTokenCallsWithErrorsCounter: Counter = Metrics.counter(METRIC_NAME_EID_SERVICE_REQUESTS, "method", "get_tc_token", "status", "500")
-    private val getEidInformationCallsSuccessfulCounter: Counter = Metrics.counter(METRIC_NAME_EID_SERVICE_REQUESTS, "method", "get_eid_information", "status", "200")
-    private val getEidInformationCallsWithErrorsCounter: Counter = Metrics.counter(METRIC_NAME_EID_SERVICE_REQUESTS, "method", "get_eid_information", "status", "500")
+    private val tcTokenCallsSuccessfulCounter: Counter =
+        Metrics.counter(METRIC_NAME_EID_SERVICE_REQUESTS, "method", "get_tc_token", "status", "200")
+    private val tcTokenCallsWithErrorsCounter: Counter =
+        Metrics.counter(METRIC_NAME_EID_SERVICE_REQUESTS, "method", "get_tc_token", "status", "500")
+    private val getEidInformationCallsSuccessfulCounter: Counter =
+        Metrics.counter(METRIC_NAME_EID_SERVICE_REQUESTS, "method", "get_eid_information", "status", "200")
+    private val getEidInformationCallsWithErrorsCounter: Counter =
+        Metrics.counter(METRIC_NAME_EID_SERVICE_REQUESTS, "method", "get_eid_information", "status", "500")
 
     @PostMapping(produces = [MediaType.APPLICATION_JSON_VALUE])
     @Operation(summary = "Create session as eService")
     @ApiResponse(responseCode = "200")
-    @ApiResponse(responseCode = "401", description = "Authentication failed (missing or wrong api key)", content = [Content()])
+    @ApiResponse(
+        responseCode = "401",
+        description = "Authentication failed (missing or wrong api key)",
+        content = [Content()]
+    )
     @SecurityRequirement(name = "apiKey")
     fun createSession(
         authentication: Authentication
-    ): Mono<ResponseEntity<CreateIdentificationSessionResponse>> {
+    ): ResponseEntity<CreateIdentificationSessionResponse> {
         val apiKeyDetails = authentication.details as ApiKeyDetails
-        return identificationSessionService.create(apiKeyDetails.refreshAddress!!, apiKeyDetails.requestDataGroups)
-            .map {
-                val tcTokenUrl =
-                    "${applicationProperties.baseUrl}$IDENTIFICATION_SESSIONS_BASE_PATH/${it.useIdSessionId}/$TCTOKEN_PATH_SUFFIX"
-                ResponseEntity
-                    .status(HttpStatus.OK)
-                    .contentType(MediaType.APPLICATION_JSON)
-                    .body(CreateIdentificationSessionResponse(tcTokenUrl))
-            }
-            .onErrorReturn(
-                ResponseEntity.internalServerError().body(null)
-            )
+        val session =
+            identificationSessionService.create(apiKeyDetails.refreshAddress!!, apiKeyDetails.requestDataGroups)
```

Comment:
I think we agreed on reducing the usage of !!. 
Maybe we can create a refactoring ticket for this... I don't like that we use !!, especially on top level.